### PR TITLE
* Updated queries for the current version of Metaspace on staging

### PR DESCRIPTION
**Pull request by Ren22**

### CODE REVIEW

**Comment by intsco** [comment]
@Ren22 too bad your merged this RP so fast )
I wanted to know why you replaced the default config to point at staging

File: metaspace/python-client/metaspace/sm_annotation_utils.py [hunk 1]
```diff
@@ -68,14 +68,36 @@ def listQuery(self, field_name, query, variables={}, batch_size=50000):
...
+        group {
+          id
+          name
+          shortName
+          urlSlug
+          members {
+            user {
+              name
+            }
+            group {
```
Comment (intsco) [inline metaspace/python-client/metaspace/sm_annotation_utils.py]: @Ren22 what's the use-case of returning a group with members and info about all groups of those members?

**Comment by Ren22** [comment]
@intsco ah, but that's not a problem to change it back to the normal metaspace config when the final release is ready

File: metaspace/python-client/metaspace/sm_annotation_utils.py (hunk 1 above)
Comment (Ren22) [reply metaspace/python-client/metaspace/sm_annotation_utils.py]: @intsco we might want to have this information easily available I think. 
For that I'm still going to add few properties in SMDataset class.

**Comment by intsco** [comment]
@Ren22 please delete branches after merging

**Pull request merged**